graylog2-web-interface/webpack.config.js

```diff
@@ -171,6 +171,5 @@ if (TARGET === 'test') {
 }
 
 if (Object.keys(module.exports).length === 0) {
-  console.error('Running in default mode');
```

Comment:
I think (correct me if I am wrong) you are mistaking `trace` for the log level, while `console.trace()` just logs to STDOUT including a rudimental stack trace. We removed logging to STDOUT recently, because it gets mixed in with a potential xml (ESLint) / JSON (webpack) output that is not pipeable to other tools then.